src/Box.js

```diff
@@ -4,11 +4,38 @@ import PropTypes from 'prop-types'
 
 const align = responsiveStyle('text-align', 'align')
 
+
+function boxShadow(props) {
+  const shadowColor = 'rgba(0,0,0,0.08)'
+  const shadows = [
+    `0 0 4px 0 ${shadowColor}`, 
+    `0 8px 8px 0 ${shadowColor}`, 
+    `0 16px 16px 0 ${shadowColor}`, 
+    `0 32px 32px 0 ${shadowColor}`
+  ]
+  const boxShadows = {
+    'small': {
+      'box-shadow': shadows[0]
+    },
+    'medium': {
+      'box-shadow': shadows.slice(0,2).join(", ")
+    },
+    'large': {
+      'box-shadow': shadows.slice(0,3).join(", ")
+    },
+    'xlarge': {
+      'box-shadow': shadows.join(", ")
```

Comment:
Picked up this linting warning per quoting string with double quotes in JS. Plz consider updating?
<img width="607" alt="screen shot 2017-10-10 at 6 55 14 pm" src="https://user-images.githubusercontent.com/6441326/31414593-accff082-adec-11e7-84fe-a4daa8f3e945.png">

Reply:
I'd say generally don't worry about linting since we'll likely be moving to using prettier instead – see #86